stdlib/src/utils/string_slice.mojo

```diff
@@ -298,7 +299,10 @@ struct StringSlice[is_mutable: Bool, //, origin: Origin[is_mutable].type,](
         Safety:
             `unsafe_from_utf8` MUST be valid UTF-8 encoded data.
         """
-
+        # FIXME: this is giving a lot of problems at comp time
```

Comment:
**Question** Can you file a bug with some more details, please and link to it from here?

Reply:
I'll add a mention for issue #3706